kafka-connect-mtls/docker-compose.yml

```diff
@@ -0,0 +1,80 @@
+---
+version: '2'
```

Comment:
Why not `3` or even `3.7` here?

Reply:
lazy c&p, but not important as per the playbook, right?

Reply:
yes, doesn't change anything here

Reply:
I have updated a few things here.